pipeline/clean/opuscleaner/generate_filters.py

```diff
@@ -0,0 +1,78 @@
+"""
+Generates filter config for a dataset based on defaults to use in OpusCleaner
+"""
+
+import argparse
+import json
+import os
+from typing import Dict, Optional
+
+
+def find_custom_filter(src: str, trg: str, dataset: str) -> Optional[Dict]:
+    # TODO: we'll likely need to move to a separate repo for those
```

Comment:
Suggestion: If we have follow-ups, it would be good to file them and link the issue here.

Reply:
I filed a ticket with the follow-up work https://github.com/mozilla/firefox-translations-training/issues/210. I hope we can start working on this soon.